lib/fuschia/entities/pesquisador.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Fuschia.Entities.Pesquisador do
+  @moduledoc """
+  Pesquisador Schema
+  """
+  use Fuschia.Schema
+  import Ecto.Changeset
+
+  alias Fuschia.Entities.Pesquisador
+  alias Fuschia.Entities.Universidade
+  alias Fuschia.Entities.User
```

Comment:
```suggestion
  alias Fuschia.Entities.{Pesquisador, Universidade, User}
```

dá pra fazer alias de vários módulos 😄